dbschema/media.esdl

```diff
@@ -0,0 +1,59 @@
+module default {
+  type Image extending Media::VisualMedia {
+    overloaded _type: Media::Type {
+      constraint expression on (__subject__ = Media::Type.Image)
+    }
+  }
+  
+  type Video extending Media::VisualMedia, Media::TemporalMedia {
+    overloaded _type: Media::Type {
+      constraint expression on (__subject__ = Media::Type.Video)
+    }
+  }
+  
+  type Audio extending Media::TemporalMedia {
+    overloaded _type: Media::Type {
+      constraint expression on (__subject__ = Media::Type.Audio)
+    }
+  }
+}
+
+
+module Media {
+  scalar type Type extending enum<
```

Comment:
Let's not specify a type enum here. We can use the actual `type` names for that.